src/SearchIndex/index.ts

```diff
@@ -72,15 +72,19 @@ export class SearchIndex {
       const aggRes = await cursor.toArray();
       return formatFacetMetaResponse(aggRes[0] as FacetAggRes, this.trieFacets);
     } catch (e) {
-      log.error(`Error while fetching facets: ${JSON.stringify(e)}`);
+      log.error(
+        `Error while fetching facets for query ${query}, with search property ${searchProperty}, and filters ${filters} ${JSON.stringify(
```

Comment:
Testing locally, it seems like we just print `[object Object]` when printing out objects like `query`. I think it might make sense to use `rawQuery` or stringify any objects we expect to print